docs/_static/nav-config.js

```diff
@@ -0,0 +1,20 @@
+const navConfig = {
+    "navTextLinks": [
+        {
+            "title": "API docs",
+            "href": "index.html",
+        },
+        {
+            "title": "Manual",
+            "href": "manual/manual_index.html",
```

Comment:
By the way these links are configurable (at least with this version of quantinuum-sphinx) 

feel free to suggest other places for the navbar to go